### DISCUSSION TITLE
Image size not showing

### DISCUSSION THREAD

**Discussion by FinnMike** [comfyanonymous/ComfyUI#5130] [Q&A]
I have a problem with ComfyUI, none of the nodes that are supposed to show the width and hight of an image seem to work. If I use the width and hight output of the node for scaling another image it works, but I am unable to show the data in the workflow.

I have tried "Easy-Use: Get Image Size and count" , "WAS: Image Size and Number", "JPS-nodes: Get Image Size". I have connected the output to "show int", "show Any", "Show Float".

All outputs work when I use them for resizing another image, but nothing shows me the width and hight in the workflow. What am I doing wrong? Is there some settings in the manager that I have missed??

**Comment by LukeG89** [comment]
I tested Was Image Size to Number and I can see all the values. To show them I used `Display Int (rgthree)` and `Display Any (rgthree)` from [rgthree-comfy](https://github.com/rgthree/rgthree-comfy).
Hope that helps!

![testing_display_sizes](https://github.com/user-attachments/assets/8a543957-7bde-418c-9a89-6cf98dd37a72)

**Reply by FinnMike** [reply]
Thank you for replying. I made the same workflow as you, but for me it doesn´t work.

![image](https://github.com/user-attachments/assets/552d29e4-3f3f-4d0f-b926-ac1b17745558)

**Reply by LukeG89** [reply]
Are your ComfyUI and custom nodes all updated? If so, probably there's still a conflict with one of your custom nodes. For example in your image I see a graphical bug in the rgthree nodes, they should look like mine.
Start by opening browser console to see if there are conflicts from a specific node. If you still can't find it, you should try disable one by one your nodes.
